main.py

```
@@ -636,28 +645,18 @@ def create_reading_order(self):
         self.stack.setCurrentWidget(self.order_widget)
 
-    def open_order_editor(self, id: int, name: str):
-        self.order_editor = ReadingOrderEditor(id, name)
-        self.order_editor.setWindowFlag(Qt.WindowType.Window)
-        # order_editor.show()
-        self.order_editor.showFullScreen()
+    def order_search(self, text: str):
+        display_info = text_search(text)
+        if display_info is None:
+            # TODO: Need to add logic here.
+            raise ValueError("Incorrect type passed!")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)